src/assets/i18n/fr.json

```diff
@@ -120,7 +123,143 @@
     },
     "add": "Ajoutez",
     "type": "Type:",
-    "features_added_soon": "Ajouté plus tard à ce forfait"
+    "features_added_soon": "Ajouté plus tard à ce forfait",
+    "create_domain": "Create domain",
+    "add_domain": "Add Domain",
+    "domain": "Domain",
+    "current_status": "Current Status",
+    "status_verification": "Verification",
+    "status_mx": "MX",
+    "status_spf": "SPF",
+    "status_dkim": "DKIM",
+    "status_dmarc": "DMARC",
+    "status_addresses": "Addresses",
+    "next": "Next",
+    "verify": "Verify",
+    "add_domain_step": {
+      "name": "Domain name",
+      "info": "Add a domain that you own to your CTemplar account."
+    },
+    "verify_step": {
+      "name": "Verify",
+      "info": "For security reasons, we need to verify that you are the owner of your domain. Please add the following code into your DNS. Do not remove it even after successful verification. This can typically be done in the control panel of your domain name registrar.<br/><br/>Please add the following TXT record:",
```

Comment:
This is a test.  Please let me know if this works and disregard this comment in github